sqlite/sqlite.spec

```diff
@@ -50,11 +59,13 @@ sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
 sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
 
 make %{?_smp_mflags}
+make %{?_smp_mflags} sqlite3_analyzer
```

Comment:
Will not work, because sqlite3_analyzer need tcl and `%configure` line 52 is call with `--disable-tcl`.

Reply:
Ok thanks. I'll fix that someday.